File: YouboraLib/persistence/entity/YBEvent.m
//
//  YBEvent.m
//  YouboraLib iOS
//
//  Created by Enrique Alfonso Burillo on 11/01/2018.
//  Copyright © 2018 NPAW. All rights reserved.
//

#import "YBEvent.h"

@implementation YBEvent

+ (NSString *)primaryKey {
    return @"id";
}

+ (NSDictionary *)defaultPropertyValues {
    return @{@"dateUpdate" : [NSDate date]};
}

@end

File: YouboraLib/persistence/entity/YBEvent.swift
//
//  YBEvent.swift
//  YouboraLib iOS
//
//  Created by Tiago Pereira on 02/05/2020.
//  Copyright © 2020 NPAW. All rights reserved.
//

import Foundation

@objcMembers public class YBEvent: NSObject {
    public let id: Int
    public let jsonEvents: String
    public let dateUpdate: Double
    public let offlineId: Int
    
    init(id: Int, jsonEvents: String, dateUpdate: Double, offlineId: Int) {
        self.id = id
        self.jsonEvents = jsonEvents
        self.dateUpdate = dateUpdate
        self.offlineId = offlineId
    }
}
